bundles/CoreBundle/EventListener/Frontend/CookiePolicyNoticeListener.php

```diff
@@ -298,13 +309,69 @@ protected function getTranslations($locale)
                 'lv' => 'Sapratu',
                 'lt' => 'Supratau',
                 'ro' => 'Am înțeles',
+            ],
+            'no' => [
+                'en' => 'No',
+                'de' => 'Nein',
+                'it' => 'No',
+                'fr' => 'No',
+                'nl' => 'Nee',
+                'es' => 'Ningún',
+                'zh' => '没有',
+                'no' => 'Ikke',
+                'hu' => 'Nincs',
+                'sv' => 'Ingen',
+                'fi' => 'Ei',
+                'da' => 'Ingen',
+                'pl' => 'Nie',
+                'cs' => 'žádný',
+                'sk' => 'žiadny',
+                'pt' => 'não',
+                'hr' => 'Ne',
+                'sl' => 'no',
+                'sr' => 'не',
+                'ru' => 'нет',
+                'bg' => 'не',
+                'et' => 'ei',
+                'el' => 'όχι',
+                'lv' => 'nē',
+                'lt' => 'ne',
+                'ro' => 'nu',
+            ],
+            'header' => [
+                'en' => 'Cookies',
+                'de' => 'Cookies',
+                'it' => 'Cookies',
+                'fr' => 'Cookies',
+                'nl' => 'Cookies',
+                'es' => 'Cookies',
+                'zh' => 'Cookies',
+                'no' => 'Cookies',
+                'hu' => 'Cookies',
+                'sv' => 'Cookies',
+                'fi' => 'Cookies',
+                'da' => 'Cookies',
+                'pl' => 'Cookies',
+                'cs' => 'Cookies',
+                'sk' => 'Cookies',
+                'pt' => 'Cookies',
+                'hr' => 'Cookies',
+                'sl' => 'Cookies',
+                'sr' => 'Cookies',
+                'ru' => 'Cookies',
+                'bg' => 'Cookies',
+                'et' => 'Cookies',
+                'el' => 'Cookies',
+                'lv' => 'Cookies',
+                'lt' => 'Cookies',
+                'ro' => 'Cookies',
             ]
         ];
 
         $translations = [];
 
         if ($this->getTranslator()) {
-            foreach (['text', 'linkText', 'ok', 'linkTarget'] as $key) {
+            foreach (['text', 'linkText', 'ok', 'linkTarget', 'header', 'no'] as $key) {
```

Comment:
Perhaps use `foreach (array_keys($defaultTranslations) as ...` to avoid hard-coding the strings twice?

Reply:
"linkTarget" is not in the previous array, but that could be fixed. But your suggestion gave me a better idea. Instead of hard-coding the strings here we could just have a class variable with an array that contains all the keys needed from shared translations.
So if you need more text bricks for your template you could just use a setter and add all the keys needed.

Reply:
Missed that one, sounds good! 👍